megamek/src/megamek/common/weapons/ASEWMissileWeaponHandler.java

```diff
@@ -0,0 +1,140 @@
+/**
+ * MegaMek - Copyright (C) 2005 Ben Mazur (bmazur@sev.org)
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License as published by the Free
+ *  Software Foundation; either version 2 of the License, or (at your option)
+ *  any later version.
+ *
+ *  This program is distributed in the hope that it will be useful, but
+ *  WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ *  or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ *  for more details.
+ */
+package megamek.common.weapons;
+
+import java.util.Vector;
+
+import megamek.common.Building;
+import megamek.common.Dropship;
+import megamek.common.Entity;
+import megamek.common.IGame;
+import megamek.common.Jumpship;
+import megamek.common.Report;
+import megamek.common.ToHitData;
+import megamek.common.Warship;
+import megamek.common.actions.WeaponAttackAction;
+import megamek.server.Server;
+
+/**
+ * @author MKerensky
+ */
+public class ASEWMissileWeaponHandler extends ThunderBoltWeaponHandler {
+
+    /**
+     *
+     */
+    private static final long serialVersionUID = 6359291710822171023L;
+
+    /**
+     * @param t
+     * @param w
+     * @param g
+     * @param s
+     */
+    public ASEWMissileWeaponHandler(ToHitData t, WeaponAttackAction w, IGame g,
```

Comment:
Let's actually fill out the comment.

Reply:
So umm, I inherited this from the parent handler, and I went back through all of its parents and found that the comment is never defined more than this. I don't know how it should be filled out...